balancer/catabalancer/sysstats.go

```diff
@@ -63,3 +97,52 @@ func GetSystemUsage() (SystemUsage, error) {
 
 	return systemUsage, nil
 }
+
+// Get bandwidth usage using the vnstat utility.
+// 'vnstat --json --iface en0 -tr 2.5' calculates traffic for given interface
```

Comment:
does this call take 2.5s?

Reply:
Yeah, the stats need to be averaged over a small duration. 1s is too inaccurate and 5s will block for too long. Perhaps an improvement for the future could be just getting the bandwidth estimates regularly in the background and the balancer logic can just use whatever was the last value recorded without blocking.